asabiris/output/smtp/service.py

```diff
@@ -144,3 +145,43 @@ async def send(
 			raise SMTPDeliverError("SMTP delivery failed")
 
 		L.log(asab.LOG_NOTICE, "Email sent", struct_data={'result': result[1], "host": self.Host})
+
+
+	def format_sender_info(self, email_info):
```

Comment:
Extend this method to return this tuple:  
`"John Doe <johndoe@example.com>", "johndoe@example.com"`

First is "user-friendly" representation, the second is email.
The "email" will be used in `To` etc.